docs/SUMMARY.md

```diff
@@ -0,0 +1,5 @@
+* [Introduction](../README.md)
+* [API](./API.md)
+* [Developer quickstart](./Quickstart.md)
+* [Project setup](./Setup.md)
+* [Integration guide](./2017-11-28-Enhancing+Your+iOS+Application+With+Salmove+SDK.md)
```

Comment:
How's integration guide different from project setup? Should be clear from the header.